src/Sql/dbo/Functions/UserCollectionDetails.sql

```diff
@@ -3,15 +3,8 @@ RETURNS TABLE
 AS RETURN
 SELECT
     C.*,
-    CASE
-        WHEN
-            OU.[AccessAll] = 1
```

Comment:
We can't lose the `AccessAll` flag in this switch, in this scenario for `ReadOnly` we're going to need to keep a switch statement and work in the `COALESCE` of the ReadOnly flag to replace the additional `OR` conditions for that flag in the `CASE` statement.

Reply:
Due to how the joins are structured later on we should be able to safely skip the `AccessAll `flag, since CU = null and CG = null if the user has `AccessAll` flag. But I agree that we should include it to make it more explicit.

Reply:
Please be explicit with `AccessAll`.